pkg/ddl/ddl_history.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb/pkg/ddl/logutil"
+	_ "github.com/pingcap/tidb/pkg/ddl/notifier" // find cycle import
```

Comment:
Can I ask what you mean by "find cycle import"? Is it used to avoid(prevent) cycle imports here?

Reply:
We can tell in future when DDL use this package, there's no cycle import. Maybe "reveal cycle import in advance"?